src/lib/shared/ui/snackbar/types.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface SnackbarOptions {
+  text: string;
+  buttonText: string;
+  callback: () => void;
```

Comment:
Also just `callback` is too generic. It only gets called when the close button is clicked, so how about `closeCallback` or so?